homeassistant/components/knx/__init__.py

```diff
@@ -208,13 +212,21 @@ def connection_config_tunneling(self):
         gateway_ip = self.config[DOMAIN][CONF_KNX_TUNNELING].get(CONF_HOST)
         gateway_port = self.config[DOMAIN][CONF_KNX_TUNNELING].get(CONF_PORT)
         local_ip = self.config[DOMAIN][CONF_KNX_TUNNELING].get(CONF_KNX_LOCAL_IP)
+        auto_reconnect = self.config[DOMAIN][CONF_KNX_TUNNELING].get(
```

Comment:
Please use `dict[key]` for optional config keys with a default value.

Reply:
Done